database/sql/V089__add_creation_timestamp_to_appdata_and_orderquotes.sql

```diff
@@ -0,0 +1,16 @@
+-- This adds `creation_timestamp` column to `app_data`, `order_quotes`
+-- in order to help data ingestion for our analytics team
+
+-- Add the columns as nullable first
```

Comment:
Can also be delete by now. The columns are never nullable with the sentinel value approach.
```suggestion
```